### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by anuarshaidenov**
## Milestone details:
- Added and styled navigation bar.
- Added and styled the content sections.
- Implemented the displaying functionality for each content section.
- Displaying the current date using Luxon library.
### As well as:
- - Displaying current links.
- - Displaying messages for better UX.
- - Changing content on `hashchange` event.
- - Made the page responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,18 +9,53 @@
...
+            <li class="nav__item">
+              <a href="#" data-link="home" class="current">List</a>
+            </li>
+            <li class="nav__item">
+              <a href="#add" data-link="add">Add new</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contact" data-link="contact">Contact</a>
+            </li>
+          </ul>
```
Comment (Emmyn5600) [inline index.html:42]: - [ ] Kindly implement the navbar sections and make sure it is working on the GitHub link demo, refer to the [design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements)

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @anuarshaidenov and @KingsleyIbe 

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:

### Changes requested.

- [ ] Check the comment under the review.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (anuarshaidenov) [reply index.html]: Hi, @Emmyn5600,
I'm not sure if I understood but I'm guessing you were referring to the lines in between the link.
If so, I've done that, although in the project requirements it is not mandatory to 100% follow the design.

**Review by od-c0d3r — APPROVED** [review]
Asslam Alaikum team @KingsleyIbe @anuarshaidenov  ,😊

Hi👋🏽, 
Awesome design :boom: :ok_hand: 
Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

## To mention : :information_source: 
 - Kindly in the next mile stone be sure to use `pair-programing` as its one of the requirements of the project. [LINK](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#:~:text=you%20need%20to%20use%20pair-programming%20for%20this%20project.%20we%20will%20check%20to%20see%20if%20both%20partners%20submitted%20code%20by%20looking%20at%20the%20commit%20history%20for%20each%20project.%20therefore%20you%20need%20to%20use%20only%20one%20repository%20for%20your%20pair-programming%20group.)
   - > index

Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**